core/src/lib.rs

```diff
@@ -246,6 +250,19 @@ impl Display for Link {
     }
 }
 
+// This is redefined here instead of importing from `beacon_types` as the ligthhouse version
+// uses #[serde(with = "serde_utils::quoted_u64")] which prevents non-json serialization
+#[derive(
+    Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize, tree_hash_derive::TreeHash,
+)]
+pub struct BeaconBlockHeader {
```

Comment:
Unfortunately, we can't just use the lighthouse version; we need this duplication. Because of this, even if we only add the slot to the journal, I'd prefer to use another Multiproof.